cmd/gaia/app/genesis.go

```diff
@@ -88,8 +88,12 @@ type GenesisAccount struct {
 	DelegatedVesting sdk.Coins `json:"delegated_vesting"` // delegated vesting coins at time of delegation
 	StartTime        int64     `json:"start_time"`        // vesting start time (UNIX Epoch time)
 	EndTime          int64     `json:"end_time"`          // vesting end time (UNIX Epoch time)
+
+	// module account fields
+	Module string `json:"module"` // name of the module
```

Comment:
Indentation seems funky. Running `gofmt -w -s cmd/gaia/app/genesis.go` should do it